app/assets/javascripts/qsets.js

```diff
@@ -41,6 +41,7 @@ function initQuestionDisplayModal($modal, $question_link) {
 
   initAnswerButton();
   initUserFeedback();
+  initVote();
```

Comment:
You'd need this if you want to make voting work on the question modal, but currently I don't see any code in _modal_question.erb for displaying votes and the upvote/downvote buttons?